ui/lib/kv/addon/components/page/secret/paths.hbs

```diff
@@ -0,0 +1,66 @@
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @pageTitle={{@path}}>
+  <:tabLinks>
+    <LinkTo @route="secret.details" data-test-secrets-tab="Secret">Secret</LinkTo>
+    <LinkTo @route="secret.metadata.index" data-test-secrets-tab="Metadata">Metadata</LinkTo>
+    <LinkTo @route="secret.paths" data-test-secrets-tab="Paths">Paths</LinkTo>
+    {{#if @secret.canReadMetadata}}
+      <LinkTo @route="secret.metadata.versions" data-test-secrets-tab="Version History">Version History</LinkTo>
+    {{/if}}
+  </:tabLinks>
+</KvPageHeader>
+
+<h2 class="title is-5 has-bottom-padding-s has-top-margin-l">
+  Paths
+</h2>
+
+<div class="box is-fullwidth is-sideless is-paddingless is-marginless">
+  <InfoTableRow
+    @label="API path"
+    @labelWidth="is-one-third"
+    @helperText="Use this path when referring to this secret in API"
+  >
+    <CopyButton
```

Comment:
Yep - Malina's work will be merged in 1.16. Her and I discussed updating her sidebranch when she was close to wrapping up the copy button swaps so that she could update any newly added buttons. 

I was going to leave a comment about which hds component to use (button vs snippet) - thanks for the reminder!